framework/filters/AccessRule.php

```diff
@@ -32,6 +32,7 @@ class AccessRule extends Component
     public $actions;
     /**
      * @var array list of the controller IDs that this rule applies to. Each controller ID is prefixed with the module ID (if any).
+     * The comparison uses [[Controller::uniqueId]], so the items in $controller look like ['shop/product'].
```

Comment:
This is only the case if the controller is located inside of a module, that should be more clear. How about the following?

> @var array list of the controller IDs that this rule applies to.
> The comparison uses [[\yii\base\Controller::uniqueId]], so each controller ID is prefixed with the module ID (if any). So for a `product` controllers in the application, you would specify this like `['product']` and if that controller is located in a `shop` module, this would be `['shop/product']`.